stock_operating_unit/security/stock_security.xml

```diff
@@ -118,4 +118,20 @@
         <field eval="0" name="perm_create"/>
     </record>
 
+    <record id="ir_rule_stock_move_line_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="stock.model_stock_move_line"/>
+        <field name="domain_force">['|',
```

Comment:
This domain is (A|B) & (C|D)? If so it's ok

Reply:
Yes.